src/views/Profile/index.jsx

```
@@ -54,9 +55,21 @@ import {
+import {
+  DeleteLearningHobbies,
+  EditLearningHobbies,
+  LearningHobbiesPlayer,
+} from "../../services/learninghobbies";
+import {
+  DeleteMondo,
+  EditMondoItem,
+  MondoDetailsViewer,
+} from "../../services/mondomarket";
 import Modal from "../../components/modal/Modal";
 import { use } from "react";
```

ESLint

[error] 'use' is defined but never used.

(no-unused-vars)